_posts/2021-12-09-debezium-1.8-cr1-released.adoc

```diff
@@ -0,0 +1,67 @@
+---
+layout: post
+title:  Debezium 1.8.0.CR1 Released
+date:   2021-12-09
+tags: [ releases, mysql, postgres, sqlserver, cassandra, oracle, db2, vitess, outbox ]
+author: gmorling
+---
+
+I'm very excited to announce the release of Debezium *1.8.0.CR1*!
+
+As were near the final release due out next week, this release focused heavily on bugfixes.
+Yet this release includes incremental snapshot support for MongoDB!
+Overall, not less than https://issues.redhat.com/issues/?jql=project%20%3D%20DBZ%20AND%20fixVersion%20%3D%201.8.0.CR%20ORDER%20BY%20component%20ASC[34 issues] have been fixed for this release.
+
+Let's take a closer look at some of them.
+
++++<!-- more -->+++
+
+== MongoDB incremental snapshots
+
+Incremental snapshotting is a feature that we first introduced as a part of Debezium 1.6 nearly six ago.
+The goals of incremental snapshots is to primarily address to very common user pain-points:
+
+* the necessity to execute consistent snapshots before streaming can begin upon connector restart
+* inability to trigger full or partial snapshots after connector has begun streaming
+
+The implementation of this feature is based on a novel approach to snapshotting originally invented by https://arxiv.org/pdf/2010.12597v1.pdf[DBLog Framework].
+Debezium implementation is described in more detail in the https://github.com/debezium/debezium-design-documents/blob/main/DDD-3.md[design document].
+
+With this release, we're excited to finally debut this feature for MongoDB.
+All Debezium core connectors now support this feature; an amazing milestone!
+
+I'd like to thank our very own https://github.com/jpechane[Jiri Pechanec] and https://github.com/kgalieva[Kate Galieva] from Shopify for their amazing efforts these last few months at refining and delivering on this feature as well as the entire community for testing and offering solid feedback.
+
+== Further fixes
+
+With 1.8 Final release scheduled for next week,
+a vast majority of the changes in this release focus on stability and bugfixes.
+Some resolved issues include:
+
+* [PostgreSQL] Parallel write can be lost during catch-up phase (https://issues.redhat.com/browse/DBZ-2792[DBZ-2792]).
```

Comment:
I don't think that's worth mentioning, given the "fix" was to remove the flaky test :)